src/js/store/orgs/actions.ts

```diff
@@ -107,13 +119,14 @@ export const provisionOrg =
     const state = getState();
     /* istanbul ignore else */
     if (isCurrentUser(originating_user_id, state)) {
-      let msg = i18n.t('Successfully created scratch org.');
-      const { name, parent } = getOrgParent(model, state);
+      const { name, parent, orgType } = getOrgParent(model, state);
+      let msg = i18n.t('Successfully created {{orgType}}.', { orgType });
+
       /* istanbul ignore else */
       if (name && parent) {
         msg = i18n.t(
-          'Successfully created scratch org for {{parent}} “{{name}}”.',
-          { parent, name },
+          'Successfully created {{orgType}} for {{parent}} “{{name}}.”',
```

Comment:
Heh, good catch. We may want to make this change consistent throughout the app? I'll add a note to https://trello.com/c/J04dg5Yi/293-i-want-consistent-capitalization-of-core-objects-terms-eg-project-task-scratch-org